packages/db/tests/lib/test-QuerySearchClient.js

```diff
@@ -96,15 +96,15 @@ test('QuerySearchClient.shift() returns next record from current set of results
     await querySearchClient.shift(),
     {
       ...records[0],
-      granule_cumulus_id: Number.parseInt(records[0].granule_cumulus_id, 10),
```

Comment:
I'm curious why the change to BigInt would make the need for Number and parseInt to go away - does the change from Int to BigInt mean the value is being grabbed as a number now instead of a string?

Reply:
Bigint column returns as a string from knex query. files.granule_cumulus_id now is a bigint, not a int anymore.

Reply:
ok. sorry for my confusion, but so now granule_cumulus_id will be a string and granule_id will also be a string, yes?

Reply:
granules.granule_id is always a string.
files.granule_cumulus_id is changed from int to bigint which matches the granule.cumulus_id, and knex returns bigint as string.
`granule_cumulus_id: records[0].granule_cumulus_id` are both string now.